hack/packages/package-tools/cmd/package-bundle-generate.go

```diff
@@ -0,0 +1,217 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
```

Comment:
Looks like the linter still didn't run on these files for some reason. Not sure why yet. But one thing you will need to change is drop the use of `io/ioutil`. That module was deprecated in Go 1.16.